pkgs/development/node-packages/overrides.nix

```diff
@@ -341,23 +341,16 @@ final: prev: {
     src = fetchFromGitHub {
       owner = "svanderburg";
       repo = "node2nix";
-      rev = "026360084db8a27095aafdac7125d7f1a93046c8";
-      sha256 = "sha256-zO/xGG10v7HGv58RLX5SFd7QOXAL2vRxCRM8IfRZ8JA=";
+      rev = "315e1b85a6761152f57a41ccea5e2570981ec670";
+      sha256 = "sha256-8OxTOkwBPcnjyhXhxQEDd8tiaQoHt91zUJX5Ka+IXco=";
     };
     nativeBuildInputs = [ pkgs.buildPackages.makeWrapper ];
     postInstall = let
       patches = [
-        # Needed to fix Node.js 16+ - PR svanderburg/node2nix#302
+        # fix nodejs attr names
         (fetchpatch {
-          name = "emit-lockfile-v2-and-fix-bin-links-with-npmv7.patch";
-          url = "https://github.com/svanderburg/node2nix/commit/375a055041b5ee49ca5fb3f74a58ca197c90c7d5.patch";
-          hash = "sha256-uVYrXptJILojeur9s2O+J/f2vyPNCaZMn1GM/NoC5n8=";
-        })
-        # Needed to fix packages with DOS line-endings after above patch - PR svanderburg/node2nix#314
-        (fetchpatch {
-          name = "convert-crlf-for-script-bin-files.patch";
-          url = "https://github.com/svanderburg/node2nix/commit/91aa511fe7107938b0409a02ab8c457a6de2d8ca.patch";
-          hash = "sha256-ISiKYkur/o8enKDzJ8mQndkkSC4yrTNlheqyH+LiXlU=";
```

Comment:
@SuperSandro2000, was this second patch to node2nix removed intentionally or in error?

It is absolutely still needed

Reply:
Yeah, that was an error from me. #228545